source/CardGames.py

```diff
@@ -138,4 +145,28 @@ def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
     for player in players:
       for _ in range(numCards):
         player.addCard(deck.getCard())
-    return True
\ No newline at end of file
+    return True
+    print(player)
+
+  
+
+
+  def printAllPlayerCards(self):
+    TotalPlayers = [player1, player2, player3]
+    for i in TotalPlayers:
+      self.printPlayerCards(i)
+
+deck1 = Deck()
+player1 = Player("john")
+player2 = Player("jeff")
+player3 = Player("george")
+dealer1 = Dealer()
+
+
+dealer1.dealCards(2, [player1], deck1)
+dealer1.dealCards(3, [player2], deck1)
+dealer1.dealCards(4, [player3], deck1)
+dealer1.printAllPlayerCards()
```

Comment:
Testing should be done in a separate file for PyTest and not in the main CardGames.py